features/steps/wishlist_steps.py

```diff
@@ -42,6 +42,11 @@ def step_impl(context, url, id):
     context.resp = context.app.get(target_url)
     assert context.resp.status_code == 200
 
+@then(u'I should see a wishlist with id "{id}"')
```

Comment:
This would be still an issue if you use this:
Then I should see a wishlist with id 4 (Even when 4 doesn't exist).